### PR TITLE
feat(translations): Add Ukrainian translation

### PR DESCRIPTION
**Pull request by Kietzmann**
#439

### CODE REVIEW

File: src/auth/screens/language-settings.js [hunk 1]
```diff
@@ -59,4 +59,9 @@ export default [
     emojiCode: ':flag-es:',
     name: 'Español',
   },
+  {
+    code: 'ua',
```
Comment (machour) [inline src/auth/screens/language-settings.js]: Use the ISO 639-1 form here: 'uk'

File: src/auth/screens/language-settings.js [hunk 2]
```diff
@@ -59,4 +59,9 @@ export default [
...
+    emojiCode: ':flag-ua:',
```
Comment (machour) [inline src/auth/screens/language-settings.js]: 'ua' is correct here. Keep it :)

File: src/locale/languages/uk.js [hunk 3]
```diff
@@ -0,0 +1,343 @@
+export const ru = {
```
Comment (machour) [inline src/locale/languages/uk.js]: this should be `uk`

File: src/auth/screens/language-settings.js (hunk 1 above)
Comment (Kietzmann) [reply src/auth/screens/language-settings.js]: resolved

File: src/locale/languages/uk.js (hunk 3 above)
Comment (Kietzmann) [reply src/locale/languages/uk.js]: resolved

File: src/auth/screens/language-settings.js (hunk 2 above)
Comment (Kietzmann) [reply src/auth/screens/language-settings.js]: resolved

**Comment by Kietzmann** [comment]
@machour 
Updated

**Comment by machour** [comment]
@Kietzmann thank you for the updates.
The travis build catched some more errors:

```
/home/travis/build/gitpoint/git-point/src/locale/languages/uk.js
  295:26  error  Unnecessary escape character: \`  no-useless-escape
  297:18  error  Unnecessary escape character: \`  no-useless-escape
```

Could you fix those too?

**Comment by Kietzmann** [comment]
@machour 
Fixed

**Comment by machour** [comment]
@Kietzmann for apostrophes, you could have used double quotes for the sentence instead, but that's fine for me. 😉 

Thank you again for your contribution!

File: src/locale/languages/uk.js [hunk 4]
```diff
@@ -0,0 +1,343 @@
...
+    },
+  },
+  search: {
+    main: {
+      repositoryButton: 'Репозиторії',
+      userButton: 'Користувачі',
+      searchingMessage: 'Пошук по {{query}}',
+      searchMessage: 'Пошук {{type}}',
+      repository: 'репозиторіїв',
+      user: 'користувачів',
```
Comment (chinesedfan) [inline src/locale/languages/uk.js]: noUsersFound, noRepositoriesFound

File: src/locale/languages/uk.js [hunk 5]
```diff
@@ -0,0 +1,343 @@
...
+      noOpenIssues: 'Не знайдено відкритих задач!',
+      noClosedIssues: 'Не знайдено закритих задач!',
+    },
+    pullList: {
+      title: 'Pull-запити',
+      openButton: 'Відкрито',
+      closedButton: 'Закрито',
+      searchingMessage: 'Пошук по {{query}}',
+      noOpenIssues: 'Не знайдено відкритих pull-запитів!',
+      noClosedIssues: 'Не знайдено закритих pull-запитів!',
```
Comment (chinesedfan) [inline src/locale/languages/uk.js]: These 2 keys should be fixed as noOpenPulls/noClosedPulls.

File: src/locale/languages/uk.js [hunk 6]
```diff
@@ -0,0 +1,343 @@
...
+      assignYourselfButton: 'Призначити самому собі',
+      assigneesTitle: 'ВІДПОВІДАЛЬНІ',
+      actionsTitle: 'ДІЇ',
+      unlockIssue: 'Розблокувати {{issueType}}',
+      lockIssue: 'Блокувати {{issueType}}',
+      closeIssue: 'Закрити {{issueType}}',
+      reopenIssue: 'Відкрити знову {{issueType}}',
+      areYouSurePrompt: 'Ви впевнені?',
+      applyLabelTitle: 'Додати мітку до цієї задачі',
+    },
```
Comment (chinesedfan) [inline src/locale/languages/uk.js]:
```
    comment: {
      commentActions: 'Comment Actions',
      editCommentTitle: 'Edit Comment',
      editAction: 'Edit',
      deleteAction: 'Delete',
    },
```

File: src/locale/languages/uk.js [hunk 7]
```diff
@@ -0,0 +1,343 @@
...
+    orgs: 'ОРГАНІЗАЦІЇ',
+    noOrgsMessage: 'Нема організацій',
+    info: 'ІНФОРМАЦІЯ',
+    company: 'Компанія',
+    location: 'Місцезнаходження',
+    email: 'Електронна пошта',
+    website: 'Сайт',
+    repositories: 'Репозиторіїв',
+    pullRequest: 'pull-запит',
+    issue: 'задача',
```
Comment (chinesedfan) [inline src/locale/languages/uk.js]: pullRequest/issue can be removed.

File: src/locale/languages/uk.js (hunk 4 above)
Comment (Kietzmann) [reply src/locale/languages/uk.js]: Resolved

File: src/locale/languages/uk.js (hunk 6 above)
Comment (Kietzmann) [reply src/locale/languages/uk.js]: Resolved

File: src/locale/languages/uk.js (hunk 7 above)
Comment (Kietzmann) [reply src/locale/languages/uk.js]: Resolved

**Comment by Kietzmann** [comment]
@chinesedfan 
Resolved

**Comment by Kietzmann** [comment]
@machour 
I ran `yarn contributors:add` and received
`ERROR: [Errno 2] No such file or directory: 'contributors:add'`
Could you help me with it? Thanks

**Comment by machour** [comment]
@Kietzmann did you initially run `yarn` ?

Try like this:
```
yarn
yarn contributors:add
```

When it asks you "What are the contribution types", select "Translations".
Let me know if it worked

File: src/locale/languages/uk.js [hunk 8]
```diff
@@ -0,0 +1,349 @@
...
+      deleteAction: 'Видалити',
+    },
+    main: {
+      assignees: 'Відповідальні',
+      mergeButton: 'Приняти pull-запит',
+      noDescription: 'Немає опису.',
+      lockedCommentInput:
+        'Заблоковано, але ви все ще можете прокоментувати...',
+      commentInput: 'Додати коментар...',
+      commentButton: 'Відправити',
```
Comment (chinesedfan) [inline src/locale/languages/uk.js]: Please delete `commentButton`. Sorry I didn't notice this one before.

Comment (Kietzmann) [reply src/locale/languages/uk.js]: Resolved

**Comment by Kietzmann** [comment]
@machour 
It was issue with cmdtest on Ubuntu 17.04. I've resolved it.

**Comment by machour** [comment]
@Kietzmann perfect! Were you able to run the app and check your translations in it?

**Comment by Kietzmann** [comment]
@machour 
Could you please provide some command or manual how to run it? 
Thanks

**Comment by machour** [comment]
@Kietzmann There you go buddy: https://github.com/gitpoint/git-point/blob/master/CONTRIBUTING.md#setup

You can start at step 6. We're on [gitter](https://gitter.im/git-point/Lobby) if you encounter *any* problem

**Comment by Kietzmann** [comment]
@machour 
there are some issues while running android virtual device on ubuntu 17.04
I think that I won't have the ability to check it.

**Pull request merged**